### DISCUSSION TITLE
Duplicated data bug in AAPS 3.0.0 (v3) with Dexcom

### DISCUSSION THREAD

**Discussion by szantos** [NightscoutFoundation/xDrip#1979] [Q&A] [answered]
AAPS 3.0.0 (v3) has been released with these notes:

https://github.com/nightscout/AndroidAPS/releases/tag/3.0.0
> Important hints
> ... There is a bug in xDrip and Dexcom native mode producing duplicated data which prevents AAPS from running in Closed loop mode. Until this get fixed using BYODA in mandatory. Using BYODA is also recommended to take advantage of Dexcom back-smoothing
> 
> Preparation steps
> ... if you are using G5/G6 switch to BYODA (recommended) - you still can use xDrip but not as collector (xDrip can receive data from BYODA) or update xDrip to latest nightly version.

(1) Is this the same issue, as discussed here?
https://github.com/Navid200/xDrip/discussions/148

(2) Has this been resolved in xDrip? (if it was a bug of xDrip)

(3) If yes, AAPS release notes has to be updated, which is clearly no work for xDrip maintainers, but AAPS devs should be informed and convinced. Or xDrip will lose a lot of it's users, which would be very sad, taking into account its superiority.

**Comment by andyrozman** [comment, marked as answer]
Yes this is the same problem... I notified Milos that this was fixed already. So I imagine he will update it as soon as he can.

**Reply by szantos** [reply]
Thanks for clarification!
The current state of AAPS release notes is very confusing (and not right).
I'll also openan issue in AAPS repo based on your answer.

**Reply by Lupum1001** [reply]
@andyrozman do you happen to know what build this was fixed in? I'm currently using the latest build from the "stable" update channel (Oct 4, 2021).
https://github.com/NightscoutFoundation/xDrip/releases/tag/2021.10.04

Is there a decent Beta build that includes the fix, or would I just need to try my luck with one of the nightly ones? I'd probably stick with the older AAPS release for now if the newer versions of XDrip aren't stable enough for a daily driver.

**Reply by Navid200** [reply]
You can see [here](https://navid200.github.io/xDrip/docs/ReleaseNotes.html) that January 14, 2022, which currently happens to be an Alpha release, addresses that particular issue.

**Reply by Lupum1001** [reply]
Excellent; thank you for the quick response. 👍
